inventory/rh-sso/group_vars/identity-hosts.yml

```diff
@@ -0,0 +1,18 @@
+# User management variables
+identities:
+  targets:
+    - rh-sso
+
+  users:
+    - user_name: example
+      first_name: Example
+      last_name: atrodrig
+      email: example@user
+      enabled: true
+      email_verified: yes
+      user_groups:
```

Comment:
Do we want to include group config yet since the rest of the config hasn't been included here yet?

Reply:
+1. Agree to remove it for now.